import_export/forms.py

```diff
@@ -54,6 +60,11 @@ def __init__(self, formats, *args, **kwargs):
 
         self.fields['file_format'].choices = choices
 
+        if hasattr(settings, 'EXPORT_DEFAULT'):
+            for i, c in enumerate(choices):
+                if c[1] == settings.EXPORT_DEFAULT.lower():
```

Comment:
Same comment as above as far as not using `.lower()`.

Reply:
Same as above.